Documents HtmlAttributesAsString usage with SPA tokens (#684)
DNNCommunity/DNNDocs: content/tutorials/client-resources/index.md

[JavaScript:{ path: "/Portals/_default/Skins/MySkin/Scripts/mySliders.min.js", priority: 10001, provider: "DnnFormBottomProvider" }]
```

> If `jsname` is provided but `path` is not provided, DNN will load the pre-installed javascript library (extension).
> 
> If both `jsname` and `path` are provided, `jsname` only serves to add a comment in the resulting html (to help troubleshooting) but the file in `path` is registered.

Author: ```suggestion
```

> If both `jsname` and `path` are provided, `jsname` only serves to add a comment in the resulting html (to help troubleshooting) but the file in `path` is registered.

Author: Cool, removed

<dnn:DnnJsInclude runat="server" FilePath="https://www.google.com/recaptcha/api.js" ForceProvider="DnnPageHeaderProvider" HtmlAttributesAsString="async:async,defer:defer" />
```

> Starting with DNN v9.11.0, `htmlAttributes` can also be used in the SPA token to add custom html attributes to a script or css tag:
> ```html
> [JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.esm.js", htmlAttributes: { type: "module" } }]
> [JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.js", htmlAttributes: { nomodule: "nomodule" } }]
> [JavaScript:{ path: "https://www.google.com/recaptcha/api.js", htmlAttributes: { async: "async", defer: "defer" } }]
> [Css:{ path: "https://fonts.googleapis.com/css2?family=Oswald&display=swap" htmlAttributes: { media:"print" } }]

Author: ```suggestion
Starting with DNN v9.11.0, `htmlAttributes` can also be used in the SPA token to add custom html attributes to a script tag:
```html
[JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.esm.js", htmlAttributes: { type: "module" } }]
[JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.js", htmlAttributes: { nomodule: "nomodule" } }]
[JavaScript:{ path: "https://www.google.com/recaptcha/api.js", htmlAttributes: { async: "async", defer: "defer" } }]
```

<dnn:DnnJsInclude runat="server" FilePath="https://www.google.com/recaptcha/api.js" ForceProvider="DnnPageHeaderProvider" HtmlAttributesAsString="async:async,defer:defer" />
```

> Starting with DNN v9.11.0, `htmlAttributesAsString` can also be used in the SPA token to add custom html attributes to a script tag:
> ```html
> [JavaScript:{ path: "/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.esm.js", htmlAttributesAsString: "type:module"}]
> [JavaScript:{ path: "/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.js", htmlAttributesAsString: "nomodule:nomodule"}]
> [JavaScript:{ path: "https://www.google.com/recaptcha/api.js", htmlAttributesAsString: "async:async,defer:defer"}]
> ```
> 

Reviewer: If https://github.com/dnnsoftware/Dnn.Platform/pull/5244 is merged as-is, the format of these tokens will change:

```suggestion
Starting with DNN v9.11.0, `htmlAttributes` can also be used in the SPA token to add custom html attributes to a script tag:
```html
[JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.esm.js", htmlAttributes: { type: "module" } }]
[JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.js", htmlAttributes: { nomodule: "nomodule" } }]
[JavaScript:{ path: "https://www.google.com/recaptcha/api.js", htmlAttributes: { async: "async", defer: "defer" } }]
```

```
